components/org.wso2.carbon.identity.application.authenticator.fido2/src/main/java/org/wso2/carbon/identity/application/authenticator/fido2/cache/FIDO2Cache.java

```diff
@@ -33,7 +33,7 @@
 
     private FIDO2Cache() {
 
-        super(FIDO2_CACHE_NAME);
+        super(FIDO2_CACHE_NAME, true);
```

Comment:
true means the cache will be backed by db?

Reply:
True means temp cache is enabled and this will be persisted.